en/mkdocs.yml

```diff
@@ -115,7 +115,7 @@ nav:
         - 'Twitter': quick-starts/twitter-as-federated-authenticator-sample.md
         - 'Microsoft Azure': quick-starts/ms-azure-as-federated-authenticator-sample.md
   - 'Guides':
-      - 'Get Started': guides/before-you-start.md
+      #- 'Get Started': guides/before-you-start.md
```

Comment:
Is this intentional?

Reply:
yes as we don't have an overview page for this section